_data/projects/elliotForWater.yml

```
@@ -0,0 +1,13 @@
+name: Elliot For Water
+desc: Elliot For Water is a search engine that donates water with your searches
+site: https://ellitoforwater.com/
+tags:
+- water
+- search-engine
+- humanitarian
+- ethical
+- open-source
+- nextjs
+upforgrabs:
+  name: good first issues
```

I think this is the fix for the linter warning:

```suggestion
  name: good first issue
```